README.md

```diff
@@ -269,6 +283,14 @@ The LIVE indicator only appears during a LIVE streaming video playback. This wil
 
 > [LIVE Indicator Color](https://github.com/aol-public/OneMobileSDK-playground-ios/tree/master/NewTutorials)
 
+#### Animations customization and disabling
+
+We've added some new animations for our player controls. So in case you want to change their duration, or to turn them off, you will have that opportuinity. These animations are added only to the default controls view controller. Check our playgrounds to find out how to turn off these animations and how to change the duration.
```

Comment:
change:
+We've added some new animations for our player controls. So in case you want to change their duration, or to turn them off, you will have that opportuinity. These animations are added only to the default controls view controller. Check our playgrounds to find out how to turn off these animations and how to change the duration.

to:
+We've added some new animations to our default player controls UX. If you want to change the animation duration, or disable the animations, you can. The sample code in this example shows how to turn off these animations as well as how to change the animation duration.